lib/ansible/modules/setup.py

```diff
@@ -139,6 +141,15 @@
 
 # Display facts from Windows hosts with custom facts stored in C(C:\\custom_facts).
 # ansible windows -m ansible.builtin.setup -a "fact_path='c:\\custom_facts'"
+
+# Gathers facts for the machines in the dbservers group (a.k.a Delegating facts)
+- hosts: app_servers
+  tasks:
+    - name: Gather facts from db servers
+      ansible.builtin.setup:
+      delegate_to: "{{ item }}"
+      delegate_facts: true
+      loop: "{{ groups['dbservers'] }}"
```

Comment:
this will add the facts of the 'last' db server to all app_servers as the loop will clobber itself in the end as facts are merged.

Reply:
@bcoca What do you suggest? Thanks.

Reply:
also, what do you want the end result to be? each app_server have access to all dbserver facts? common fact? ..